.gitignore

```diff
@@ -5,6 +5,7 @@ dist/
 build
 coverage
 *.xml
+.venv
```

Comment:
I added this just for python venvs that aren't via conda. I assumed it was a standard thing to ignore for Python .gitignores.